config/initializers/simple_form/wrappers.rb

```diff
@@ -15,6 +15,13 @@
     end
   end
 
+  config.wrappers :rademade_simple, :tag => 'div', :class => 'form-box' do |form_box|
+    form_box.use :html5
+    form_box.wrapper :tag => 'div', :class => 'input-holder' do |input_holder|
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.